pkg/util/qos/qos_helper.go

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
it has nothing to do with qos at all; if you want to make this logic more general, you should move it to dir `util/metic` and arrange them into several files like `metric_numa.go`, `metric_pod.go` or something like that

Reply:
If it is defined in util/metic, it will introduce circular reference problems, so it has to be placed in another package.

Reply:
maybe it can be defined in `metaserver/agent/metric/util`, and to avoid circular reference, the function should better use `MetricsFetcher` rather than `metaServer` as paramethers